### ISSUE TITLE
The last version not working with larave 5.3

### ISSUE THREAD

**Issue by hamdallah90** [marcelgwerder/laravel-api-handler#32]
Cheers

**Comment by Jonybang** [comment]
```
RuntimeException in Facade.php line 234:
A facade root has not been set.
in Facade.php line 234
at Facade::__callStatic('parseMultiple', array(object(Builder), array('id', 'name'))) in TagController.php line 20
at TagController->index()
at call_user_func_array(array(object(TagController), 'index'), array()) in Controller.php line 55
at Controller->callAction('index', array()) in ControllerDispatcher.php line 44
at ControllerDispatcher->dispatch(object(Route), object(TagController), 'index') in Route.php line 189
at Route->runController() in Route.php line 144
at Route->run(object(Request)) in Router.php line 653

```

**Comment by marcelgwerder** [comment]
Can you show me some context? How does your setup look like? How exactly do you use the package? I have a pretty safe assumption on why you're suddenly getting this exception. However I cannot reproduce it yet, everything working fine on 5.3 for me and thus it's hard to tell whether I fixed it or not.

**Comment by marcelgwerder** [comment]
Should be fixed in v0.6.1, let me know if that works for you.
